Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -826,7 +830,7 @@ IPCCommandResult NetIPTop::HandleGetInterfaceOptRequest(const IOCtlVRequest& req
       if (RetVal == NO_ERROR)
       {
         unsigned long dwBestIfIndex = 0;
-        IPAddr dwDestAddr = (IPAddr)0x08080808;
+        IPAddr dwDestAddr = static_cast<IPAddr>default_main_dns_resolver;
```

Comment:
Sorry, I should've been clearer, but the syntax should be: `static_cast<IPAddr>(default_main_dns_resolver)`

Reply:
Oops, looks like I should actually both to setup a windows build environment too.
Committing blindly only gets so far. :P